src/charm.py

```diff
@@ -49,7 +49,7 @@ def reload(self) -> bool:
     @staticmethod
     def _get(url: str) -> Optional[dict]:
         try:
-            response = requests.get(url)
+            response = requests.get(url, timeout=2.0)
```

Comment:
What about having this `timeout` value as a constant instead of hardcoded here?

Reply:
Good point! Fixed.